CODE_OF_CONDUCT.md

```diff
@@ -4,6 +4,8 @@ We are a community of, and in solidarity with, people from every gender identity
 
 This Code of Conduct applies to every environment and platform relating to the Greenpeace Planet 4 project.
 
+Please find the [link to the Model Greenpeace Code of Conduct here](https://drive.google.com/file/d/18BvOe-2hIuRAfNvKNX5nuOnxIPwzONKj/view) & an email address of the [International Persons of Trust](conduct@greenpeace.org). 
```

Comment:
Please can you insert my short email for the time being? ltiralon@greenpeace.org